connect-examples/oauth/java/src/main/java/com/squareup/oauthexample/OAuthHandler.java

```diff
@@ -35,25 +35,27 @@
 import java.net.InetSocketAddress;
 import java.net.URI;
 import java.util.List;
+import java.util.LinkedList;
 
 import org.apache.http.NameValuePair;
 import org.apache.http.client.utils.URLEncodedUtils;
 
-import org.json.JSONObject;
-
 public class OAuthHandler {
 
 
   // FOR SANDBOX TESTING:
   // If you are testing the OAuth flow in the sandbox, use your sandbox application
   // ID and secret. You MUST also set the CONNECT_HOST to "https://connect.squareupsandbox.com"
 
-  // Your application's ID and secret, available from your application dashboard.
-  private static final String APPLICATION_ID =  "REPLACE_ME";
-  private static final String APPLICATION_SECRET =  "REPLACE_ME";
-  // The base URL for every Connect API request
-  private static final String CONNECT_HOST = "https://connect.squareup.com";
-  private static final String SCOPES = "REPLACE_ME";
+  // Options are Environment.SANDBOX, Environment.PRODUCTION, and Environment.CUSTOM
+  private static final Environment ENVIRONMENT = Environment.SANDBOX;
+  // Your application's ID and secret, available from the OAuth tab in the Developer Dashboard
+  private static final String APPLICATION_ID =  "sandbox-sq0idb-eCakpKGXFTnrQd13mL2xuQ";
+  private static final String APPLICATION_SECRET =  "sandbox-sq0csb-n64Aq7nVEwGxJbPjpwlC6JneKT-Jbh2oIPY319X4H-Q";
```

Comment:
oops, shouldn't have left it there

Reply:
It's looks like this is still in the git history. You may want to squash your commits and then force push to remove that commit. Also be sure to "replace secret" for this app just in case.